CMakeLists.txt

```diff
@@ -34,6 +35,21 @@ set(CMAKE_EXPORT_COMPILE_COMMANDS 1)
 if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
   set (CMAKE_CXX_FLAGS -fsized-deallocation)
 endif()
+
+if(coverage AND NOT MSVC)
+  set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pg -DHAVE_EXECINFO_H -g3")
+endif()
+if(testMPI)
+  function(add_phare_test binary directory)
+    add_test(NAME ${binary} COMMAND mpirun -n 2 ./${binary} WORKING_DIRECTORY ${directory})
```

Comment:
do you think the number of processes could also be a parameter of the function? (would default=2 possible in that case?)